.tflint.hcl

```diff
@@ -1,5 +0,0 @@
-plugin "aws" {
```

Comment:
Why was that file removed?

Reply:
Because our workflow-config takes care of that file, and we don't need a special tflint in this case